torchvision/models/optical_flow/raft.py

```diff
@@ -560,14 +567,19 @@ def _raft(
             multiplier=0.25,  # See comment in MaskPredictor about this
         )
 
-    return RAFT(
+    model = RAFT(
         feature_encoder=feature_encoder,
         context_encoder=context_encoder,
         corr_block=corr_block,
         update_block=update_block,
         mask_predictor=mask_predictor,
         **kwargs,  # not really needed, all params should be consumed by now
     )
+    if pretrained:
+        state_dict = load_state_dict_from_url(_MODELS_URLS[arch], progress=progress)
+        model.load_state_dict(state_dict)
```

Comment:
Nit: If you move the model loading to the legacy model builders, you will be able to reuse `_raft` without requiring any of the changes you made here. You will also avoid adding `_raft_builder`, leading overall to less code. You will need to duplicate the weight loading lines though so up to you.